server/src/main/java/io/spine/server/tuple/package-info.java

```diff
@@ -64,11 +79,28 @@
  *    <li>{@link io.spine.server.tuple.Quintet Quintet&lt;A, B, C, D, E&gt;}
  * </ul>
  *
- * <p>Basic tuple classes allow {@link java.util.Optional Optional} starting from
- * the second generic argument.
+ * <p>The first generic parameter {@code <A>} must always be a specific
+ * {@link com.google.protobuf.Message Message} class.
+ *
+ * <p>Tuple classes allow {@link java.util.Optional Optional} starting from
+ * the second generic argument. The example below shows the method which always returns
+ * the {@code TaskCreated} event, and returns the {@code TaskAssigned} event only if
+ * the {@code CreateTask} command instructs to assign the task to a person.
+ * <pre>
+ *    {@literal @}Assign
+ *     Pair&lt;TaskCreated, Optional&lt;TaskAssigned&gt;&gt; on(CreateTask cmd) { ... }
+ * </pre>
  *
  * <h1>Alternatives</h1>
  *
+ * <p>In some cases it is needed to return one message from a limited set of possible options.
+ * For example, the following method issues a command {@code MoveToTrash} if no work
+ * was reported on the task, and {@code CancelTask} if some work has been already logged.
+ * <pre>
+ *    {@literal @}Command
+ *     EitherOf2&lt;MoveToTrash, CancelTask&gt; handle(RemoveTask cmd) { ... }
```

Comment:
Can we use `{@literal }` instead of escape sequences?

Reply:
I tried. It becomes too wordy specially when it comes to links or code samples.